src/raytracer/getTgayReflectionLoss.m

```diff
@@ -0,0 +1,107 @@
+function reflectionLossdB = getTgayReflectionLoss(MaterialLibrary,...
+                materialIndex, multipath)
+% getTgayReflectionLoss returns reflection loss in dB. 
```

Comment:
missing documentation.
also a reference would be nice

Reply:
done